### DISCUSSION TITLE
Add functionality to use the phone's camera to upload a user avatar photo

### DISCUSSION THREAD

**Discussion by vanillajonathan** [community/community#52721] [Mobile]
### Select Topic Area

Product Feedback

### Body

Users might want to use the camera on the phone to take a selfie for use as avatar photo.
The GitHub Mobile app should have a function that lets the user take a photo using the device's selfie camera.

**Comment by stevepopovich** [comment]
Hi @vanillajonathan thanks for the feedback! I passed this on to the team for prioritization!
